### ISSUE TITLE
Client - Timeout on join, no messages.

### ISSUE THREAD

**Issue by jpoles1** [graarh/golang-socketio#35]
I've got a fully functional socket.io server, and I've got other clients listening to it, but this library doesn't seem to do this. It joins briefly, cannot receive messages, then times out before disconnecting. What's going on? I'm using the example client.go?

**Comment by mtfelian** [comment]
client.go is designed to work with server.go and it is only an example

**Comment by jpoles1** [comment]
I'm suggesting it's not a very good example, and could be improved. Additionally, if the client.go is only meant to work with server.go, we should say this in the docs/readme.

**Comment by mtfelian** [comment]
I mean client.go in the examples dir. You did not provide any code so what to do?..

**Comment by jpoles1** [comment]
Yes, I used client.go from the examples. I would love to provide better working code, but after a fair bit of time spent tinkering, I have been unable to get a client working.

**Comment by jpoles1** [comment]
I opened this issue so that the developers would know the code is not working 100%, and so that others running into my same issues will know it's not their setup alone that's having problems.

**Comment by jpoles1** [comment]
I also tried your version @mtfelian, and got a similar issue.

**Comment by mtfelian** [comment]
Can you please provide a code samples what you tried at server and client sides in "minimal version".
